src/Command/ProbeWorkerCommand.php

```diff
@@ -37,14 +37,14 @@ class ProbeWorkerCommand extends Command
     protected $logger;
 
     /**
-     * @var CommandFactory
+     * @var TaskFactory
      */
     private $commandFactory;
```

Comment:
Rename this one as well ?

Reply:
Good catch. I've renamed all of the $commandFactory variables to $taskFactory (or simply $factory).